.trivyignore

```diff
@@ -0,0 +1,8 @@
+# Github repository shouldn’t be public.
```

Comment:
can you add a link with the trivy related docs?

Reply:
Do you mean for each suppressed warning or trivy in general?

Reply:
in general (with a list of possible configuration)

Reply:
Updated with the main page of misconfiguration docs